docs/en/index.md

```diff
@@ -16,7 +16,9 @@ FreshRSS has a lot of features including:
 - Multi-users so you can host your friends and your family
 - And a lot more!
 
-This documentation is splitted in two sections:
+This documentation is splitted in four sections:
```

Comment:
The past tense of split is split. When combined with sections, it's *split into*.